pkg/resourceinterpreter/default/native/prune/prune.go

```diff
@@ -98,9 +98,9 @@ func removeGenerateSelectorOfJob(workload *unstructured.Unstructured) error {
 		return err
 	}
 	if exist {
-		if util.GetLabelValue(matchLabels, "controller-uid") != "" {
-			delete(matchLabels, "controller-uid")
-		}
+		delete(matchLabels, "controller-uid")
+		delete(matchLabels, batchv1.ControllerUidLabel)
```

Comment:
It'd be great if we can add a comment to explain why we need to remove the label.
same as below.